inc/sainexthopgroup.h

```diff
@@ -133,6 +192,15 @@ typedef enum _sai_next_hop_group_attr_t
      */
     SAI_NEXT_HOP_GROUP_ATTR_COUNTER_ID,
 
+    /**
+     * @brief Number of hash buckets in the group
+     *
+     * @type sai_uint32_t
+     * @flags MANDATORY_ON_CREATE | CREATE_ONLY
+     * @validonly SAI_NEXT_HOP_GROUP_ATTR_TYPE == SAI_NEXT_HOP_GROUP_TYPE_L4_SESSION_CONSISTENT
+     */
+    SAI_NEXT_HOP_GROUP_ATTR_NUM_OF_BUCKETS,
```

Comment:
Following our discussion, let's change it to SAI_NEXT_HOP_GROUP_ATTR_NUM_OF_ENTRIES, or 
SAI_NEXT_HOP_GROUP_ATTR_TABLE_SIZE or similar, to make it more generic and not imply hash-based implementation (the comment changes accordingly).
Thanks!

Reply:
SAI_NEXT_HOP_GROUP_ATTR_MAX_NUM_OF_ENTRIES?